packages/thunderstore-api/src/apiFetch.ts

```diff
@@ -100,19 +116,27 @@ export async function apiFetch(props: {
   });
 
   if (!response.ok) {
-    throw await ApiError.createFromResponse(response);
+    const apiError = await ApiError.createFromResponse(response, {
```

Comment:
Nitpick: speaking of extra variables, I think this is one and it would be clearer to just throw on this line.